.rubocop.yml

```diff
@@ -8,14 +8,6 @@ inherit_mode:
     - Exclude
     - AllowedMethods
 
-Layout/AlignHash:
```

Comment:
`AlignHash` was renamed to `HashAlignment` back in 2019.  We don't need both: https://github.com/rubocop/rubocop/commit/7ab80f9e1d262ff34fe4a1ea1a752a22bec26310